src/App.tsx

```diff
@@ -1,10 +1,76 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState, useMemo } from 'react';
+
 import { UserWarning } from './UserWarning';
+import { Header } from './components/Header';
+import { Todos } from './components/Todos';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
+
+import { getTodos } from './api/todos';
+
+import { Todo } from './types/Todo';
+import { FilteredBy } from './types/FilteredBy';
 
-const USER_ID = 0;
+const USER_ID = 9964;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[] | null>(null);
+  const [isDataLoaded, setDataLoaded] = useState(false);
+  const [filteredBy, setFilteredBy] = useState<FilteredBy>(FilteredBy.All);
+  const [isCompletedPresent, setCompletedPresent] = useState(false);
+  const [isLoadingError, setLoadingError] = useState(false);
+  let timerId = 0;
```

Comment:
Consider using `useRef` to store the `timerId` so that it does not create a new variable on each render.